en/docs/choreo-samples/explore-the-demo-organization.md

```diff
@@ -1,12 +1,12 @@
-# Explore the Demo Organization
+# Explore the demo organization
 
-The demo organization in Choreo provides a hands-on view of a fully deployed system in a read-only mode. This organization is maintained by Choreo and demonstrates a real-world use case to help you understand how Choreo simplifies developing, deploying, and managing cloud native applications.
+The demo organization in Choreo provides a hands-on view of a fully deployed system in read-only mode. This organization, maintained by Choreo, demonstrates a real-world use case to help you understand how Choreo simplifies developing, deploying, and managing cloud-native applications.
 
 ## Prerequisites
 
 - If you are signing in to the Choreo Console for the first time, create an organization as follows:  
     1. Go to the [Choreo Console](https://console.choreo.dev/) and sign in using Google, GitHub, or Microsoft.  
-    2. Enter a unique organization name, for example, `Stark Industries`.  
+    2. Enter a unique organization name (e.g., `Stark Industries`).  
```

Comment:
To adhere to the Google style guide, we generally avoid using e.g., https://developers.google.com/style/word-list#eg